src/content/h3/guides/shaders/pbr-shader/readme.md

```diff
@@ -0,0 +1,175 @@
+
+In Halo 3, the shininess/highlights of a surface is defined using Specular maps. Halo 3 uses more stylized Specular maps compared to other industry Specular maps, so you will need to create your own. Below is an *experimental* method in creating Halo 3 styled Specular maps. 
+
+You will need three textures:
+
+  - Metallic
+  - Roughness
+  - Ambient Occulsion
+
+
+Conversion steps
+
+1. Open up your Metallic map in your photo editing program.
+  * ![](F.png "Metallic map layer")
+2. Create a new layer over your Metallic map. 
+3. Copy and paste your Roughness map into the new empty layer.
+4. Press <kbd>Ctrl + I</kbd> to invert the Roughness map.
+5. Set the blend mode of the Roughness map to **Lighten**
+  * ![](G.png "Inverted Roughness Map")
+6. Create a new layer over your inverted Roughness map.
+7. Copy and paste your Ambient Occulsion map into the new empty layer.
+8. Set the blend mode of the Ambient Occulsion to **Multiply** and the opacity to **75%**.
+  * ![](H.png "Multiplied Ambient Occulsion map")
+9. Save your new Specular map as .TIF in your asset's bitmaps folder.    
+
+And there you go, you should now have a working Specular map. Make sure to save a editable copy of the Specular map in case you wish to make changes later.
+
+# Step 3: Creating a Control Map
+
+Halo 3's PBR shader needs one more texture map to complete the shader. This Control map holds two texture maps in the RGB color channels. In the red channel is the Specular map and in the green channel is the Roughness map. The blue channel is just a solid white. This is the most important step. Without the correct Control map setup, your asset will not look correct in-engine.
+
+Setup steps
+
+1. Merge all the layers of your Specular map.
+2. Copy your Specular map.
+3. Create a new image with the exact same pixel resolution of the Specular map.
+4. Open the color channels of the new image and paste the Specular map in the red channel.
+  * ![](I.png "Specular map in the red channel")
+
+  **NOTE**: *If you do not have a Metallic map, make the red channel solid black.*
+
+5. Copy your unaltered Roughness map and paste it into the green channel.
+6. Make the blue channel solid white.
+  * ![](J.png "Roughness map in the green channel")
+7. Save your new Control map as .TIF in your asset's bitmaps folder.    
+
+The final Control map should resemble a pink and blue texture, with the pink colors representing the reflective surfaces and the blue colors the dull surfaces. If you have a material that has no metal, the Control map should only have blue colors. 
+
+* ![](K.png "Final Control map")
+
+# Step 4: Converting a Normal Map
+
+Make sure to identify what type of Normal map you have. Halo 3 uses DirectX normals and those do not need to be converted to work with the engine. Simply save the normal as a .TIF in your asset's bitmaps folder and you will be good to go.
```

Comment:
maybe link to something that explains what the difference between those two is? Or would it be obvious to anyone reading this guide?